src/index.tsx

```diff
@@ -43,6 +63,14 @@ async function setupConfig() {
 trackAppState();
 enableScreens();
 
+// Adds ability to automatically animate between layout paint changes.
+if (
+  Platform.OS === 'android' &&
+  UIManager.setLayoutAnimationEnabledExperimental
+) {
+  UIManager.setLayoutAnimationEnabledExperimental(true);
+}
```

Comment:
https://github.com/facebook/react-native/issues/25832

Bit unsure about this, maybe we should feature toggle this?

Reply:
Yeah, good point. I'll fix

Reply:
Just removing it entierly. Triggering some errors